label_studio_converter/utils.py

```diff
@@ -99,45 +96,27 @@ def _get_upload_dir(project_dir=None, upload_dir=None):
     return upload_dir
 
 
-def download(url, output_dir, filename=None, project_dir=None, return_relative_path=False, upload_dir=None,
+def download(url,
```

Comment:
do we still need for this function if it's just a wrapper around `get_local_path` ?

Reply:
We need this function to process **return_relative_path** option